W3ChampionsStatisticService/Services/TrackingService.cs

```diff
@@ -45,7 +41,10 @@ public void TrackUnauthorizedRequest(string authorization, ControllerBase contro
 
     public void TrackException(Exception ex, string message)
     {
-        _logger.LogError(ex, message);
```

Comment:
We need the logger to log to grafana, don't we?

Reply:
This isn't serilog either